user-settings-example.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  console.communityProvidersWarning: "true"
+  console.guidedTour: '{"dev":{"completed":true}}'
```

Comment:
Do we need this sample?
What's the intended use?
Might be better to simply have a comment in the `useUserSettings` hook.

Reply:
Removed the example